configs/server/forseti_conf_server.yaml.in

```diff
@@ -16,6 +16,12 @@ inventory:
     # project number instead of project id, e.g. "projects/<project number>".
     root_resource_id: {ROOT_RESOURCE_ID}
 
+    # Resources to be excluded during the inventory process.
+    # Only organizations/<ORG_NUMBER>, folders/<FOLDER_NUMBER> and
+    # projects/<PROJECT_ID> are accepted. The child resources under
```

Comment:
I'm concerned that CAI at some point will allow excluded resources to be passed in, but will require the project id to be the project number, since that is what they index on. Is there any way that we can support project number instead of project id, or both id and number and translate internally? Maybe this is something that can be addressed in the future, I just hate to break people if they start using project id and then have to change in the future to project number for CAI support.